recipes/sentry-native/all/conanfile.py

```diff
@@ -16,20 +16,22 @@ class SentryNativeConan(ConanFile):
     license = "MIT"
     topics = ("conan", "breakpad", "crashpad",
               "error-reporting", "crash-reporting")
-    exports_sources = ["CMakeLists.txt"]
+    exports_sources = ["CMakeLists.txt", "patches/*"]
     generators = "cmake", "cmake_find_package"
     settings = "os", "arch", "compiler", "build_type"
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
         "backend": ["none", "inproc", "crashpad", "breakpad"],
         "transport": ["none", "curl", "winhttp"],
+        "qt": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "backend": "inproc",
-        "transport": "curl"
+        "transport": "curl",
+        "qt": False,
```

Comment:
Is qt integration by default useful?

Reply:
Yes but at the expense of current users who do no require it

https://docs.sentry.io/platforms/native/guides/qt/#install

Looks off by default from upstream

Reply:
buildroot has some kind of ci system where it builds random combinations to find regressions.
Something similar where non-default option combinations are built would be useful.